bitacross-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -162,6 +162,18 @@ where
 		}
 	});
 
+	// btc sign task data share
+	io.add_sync_method("bitacross_btcDataShare", move |params: Params| {
```

Comment:
This method simply processes any `DirectCall` - not only variants related to musig2 ceremony rounds.
For example i've tried to submit `SignEthereum` direct call - it was executed but signature was not returned.

Why you decided to add it ?

Reply:
We can't see what concrete type of `DirectCall` the request contains, because it's encrypted.

Reply:
So there is nothing specific to 'btcDataShare' except the name here.

Reply:
And different handling, `bitacross_task_create_inner `and `bitacross_data_share_inner`